docs/contributing/contributions_api.rst

```diff
@@ -30,21 +30,63 @@ repository and contribute changes back to the project master branch using pull r
 Test code
 =========
 
-Test code should be used to verify new and changed functionality. Changes to DroneKit can be tested locally by 
-rebuilding your git branch and running it against SITL. The links below provide information on how 
-to set up a development environment on your development computer:
+Test code should be used to verify new and changed functionality. Changes to DroneKit can be tested locally by rebuilding your git branch and running it locally. The links below provide information on how to set up a development environment on your development computer:
 
 * :ref:`dronekit_development_linux`
 * :ref:`dronekit_development_windows`
 
-.. note:: At time of writing (June 2015) we do not yet have formal acceptance tests (unit tests, system tests).
+Running Web Client Tests
+========================
 
-We recommend that you provide your test script as a gist, and run any the 
-:ref:`DroneKit-Python Examples <example-toc>` that are relevant.
- 
-For example, the addition of a new API item for retrieving a vehicle attribute would need test code for 
-getting, setting (if applicable) and printing the attribute. An easy way to create this would be to 
-update and run :ref:`example-vehicle-state`.
+Web client tests use nose. First export a DRONEAPI_KEY in the format `<id>.<key>`. The run:
 
+.. code:: bash
 
+    cd dronekit-python
+    nosetests tests/web
 
+Running Integrated Tests
+========================
+
+Integrated tests use the SITL environment to run DroneKit tests against a simulated copter.
```

Comment:
Again, worth a sentence to provide high level info about what our integrated tests are/how they differ from the web tests.

So basically there is a script in **dronekit-python/tests** called sitl that you run using ``python -um sitl`` to start sitl? Is this the sitl runner based?